resources/3rdparty/CMakeLists.txt

```diff
@@ -322,11 +322,16 @@ else()
 endif()
 
 if (STORM_SHIPPED_CARL)
+    if(NOT APPLE_SILICON)
```

Comment:
I find it a bit confusing that these variables are set in two places. Would it make sense to set it to `ON` by default and only disable it in the root `CMakeLists.txt`?

Reply:
Yes, that makes sense. I will change that.